iam_check/lib/reporter.py

```diff
@@ -8,19 +8,21 @@
 from iam_check.tools import regex_patterns
 
 default_finding_types_that_are_blocking = ['ERROR', 'SECURITY_WARNING']
+default_finding_codes_that_are_blocking = []
 
 
 class Reporter:
 	"""
 	Determines what findings should be reported to the end user based on parameters provided when starting validation.
 	"""
 
-	def __init__(self, findings_to_ignore, finding_types_that_are_blocking, allowed_external_principals):
+	def __init__(self, findings_to_ignore, finding_types_that_are_blocking, allowed_external_principals, finding_codes_that_are_blocking=default_finding_codes_that_are_blocking):
```

Comment:
open to not doing a kwarg here but don't want to break existing implementations before a larger release